functions/wavlen.h

```diff
@@ -1,6 +1,14 @@
 #ifndef FUNCTIONS_WAVELEN_H
 #define FUNCTIONS_WAVELEN_H
 
+// Usage: WavLen<EFFECT>
```

Comment:
You can also use just WavLen<>.
If you use that inside a TransitionEffectL, then it will automatically use the right effect.

Reply:
oh wow. ok even better.